sdk/template/azure-template/test/CMakeLists.txt

```diff
@@ -9,16 +9,18 @@ set(CMAKE_CXX_STANDARD_REQUIRED True)
 
 add_compile_definitions(AZURE_TEST_DATA_PATH="${CMAKE_BINARY_DIR}")
 
+include(GoogleTest)
```

Comment:
What would happen if we didn't have this line?

Reply:
The template project was using the old `add_gtest` command which doesn't need the `include(GoogleTest)`
For using the new command, we need to include the GoogleTest module from CMake.

I think we also add this module from the root CMake project. But we want to keep each project separated and with the option to be built alone, so, we neeed this include for each test project using gtest